src/atoms/gv-select-native.js

```diff
@@ -182,7 +182,7 @@ export class GvSelectNative extends InputElement(LitElement) {
   }
 
   getInputElement() {
-    return this.shadowRoot.querySelector('select');
+    return super.getInputElement();
```

Comment:
@ThibaudAV @gaetanmaisse It's chelou

Reply:
The implementation of this method was  the same as the one super class (our version is less protective). Maybe we should just remove the method to inherit the super one directly?

Reply:
No it's not the same